docs/data-ai/data/visualize.md

```diff
@@ -25,52 +25,143 @@ Create a dashboard visualizing numeric data from components with the **TELEOP**
 
 ### Prerequisites
 
-{{% expand "A configured machine with sensor components" %}}
+{{% expand "A machine with at least one sensor, camera, or other component that captures data" %}}
 
-Make sure your machine has at least one of the following:
+Make sure your machine has at least one configured sensor component.
```

Comment:
Looks like the expander heading was updated per Naomi's comment, but the expander contents should match:
```suggestion
Make sure your machine is configured with at least one component that can capture data, for example:

- [Sensor](/operate/reference/components/sensor/)
- [Movement sensor](/operate/reference/components/movement-sensor/)
- [Camera](/operate/reference/components/camera/)
```